.github/workflows/ci-julia-nightly.yml

```
@@ -9,20 +9,24 @@ on:
     tags: '*'
 jobs:
   test-julia-nightly:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    name: v${{ matrix.version }}/t-${{ matrix.threads }}/group-${{ matrix.group }}/${{ github.event_name }}/${{ matrix.arch }}+${{ matrix.os }}
```

```suggestion
    name: t-${{ matrix.threads }}/group-${{ matrix.group }}/${{ github.event_name }}/${{ matrix.arch }}+${{ matrix.os }}
```
Here I just removed "vnightly", my bad.